docs/tools/cli/service/privatelink.rst

```diff
@@ -0,0 +1,423 @@
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Lists PrivateLink connection information for a service.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--project``
+    - The project to fetch details for
+
+**Example:** List PrivateLink for ``kafka-12a3b4c5`` service.
+
+::
+
+    avn service privatelink aws connection list kafka-12a3b4c5
+
+An example of output:
+
+.. code:: text
+
+    {
+      "dns_name": "vpce-0123456789abc1345-qfhrjbis.vpce-svc-0abcdef0123456789.us-east-1.vpce.amazonaws.com",
+      "privatelink_connection_id": "plc39413abcdef",
+      "state": "active",
+      "vpc_endpoint_id": "vpce-0123456789abc1345"
+    }
+
+.. _avn_service_privatelink_aws_create:
+
+``avn service privatelink aws create``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Creates AWS PrivateLink for a service. To add multiple principals, repeat `--principal` parameter.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--project``
+    - The project to fetch details for
+  * - ``--principal``
+    - ARN that is allowed to connect (example: ``arn:aws:iam::123456789012:user/cloud_user``)
+  * - ``--format``
+    - Format of the output string
+
+**Example:** Create PrivateLink for ``kafka-12a3b4c5`` service.
+
+::
+
+    avn service privatelink aws create --principal 'arn:aws:iam::123456789012:user/cloud_user' kafka-12a3b4c5
+
+An example of output:
+
+
+.. code:: text
+
+    AWS_SERVICE_ID  AWS_SERVICE_NAME  PRINCIPALS                                 STATE   
+    ==============  ================  =========================================  ========
+    null            null              arn:aws:iam::123456789012:user/cloud_user  creating
+
+.. _avn_service_privatelink_aws_delete:
+
+``avn service privatelink aws delete``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Deletes PrivateLink for a service.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--project``
+    - The project to fetch details for
+  * - ``--format``
+    - Format of the output string
+
+**Example:** Delete PrivateLink for ``kafka-12a3b4c5`` service.
+
+::
+
+    avn service privatelink aws delete kafka-12a3b4c5
+
+An example of output:
+
+.. code:: text
+
+    AWS_SERVICE_ID              AWS_SERVICE_NAME                                         PRINCIPALS                                 STATE   
+    ==========================  =======================================================  =========================================  ========
+    vpce-svc-1234567890abc1234  com.amazonaws.vpce.us-east-1.vpce-svc-1234567890abc1234  arn:aws:iam::123456789012:user/cloud_user  deleting
```

Comment:
The state is `deleting`, how can one be sure it's deleted?

Reply:
Added
```
The deletion can take some time to complete. You can check the status by running ``avn service privatelink aws get``.
```